src/__tests__/Button.spec.js

```diff
@@ -12,4 +12,19 @@ describe('<Button />', () => {
     const div = document.createElement('div');
     ReactDOM.render(<Button />, div);
   });
+
+  it('should render button text', () => {
+    const btn = shallow(<Button name="8"/>);
+    expect(btn.text()).toEqual("8");
+  });
+
+  it('should execute passed function', () => {
```

Comment:
This looks good. also might want to look at using a jest mock function.  Then in your expect use something like toHaveBeenCalledWith.  https://jestjs.io/docs/en/mock-functions.